gnome-abrt.spec

```diff
@@ -62,7 +62,9 @@ provides them with convenient way for managing these problems.
 
 %build
 %meson \
-    %{!?fedora:-Dlint=false} \
```

Comment:
So why is this a problem for ELN? Shouldn’t it have all of `%eln`, `%fedora` and `%rhel` defined?

Reply:
The above bit would install the required dependencies.

Reply:
Yes, the original line could technically stay as-is. However, the above condition surrounding the BuildRequires can't. Relying upon `%{fedora}` to be `%{nil}` for ELN vs _undefined_ for Fedora is really subtle. It's best to be obvious.

To be even more obvious, the line below could be
```
%if  !(0%{?fedora} || 0%{?rhel} >= 9)
```
or
```
%if  !0%{?fedora} && (0%{?rhel} && 0%{?rhel} < 9)
```
Even more obvious would be
```
%if 0%{?fedora} || 0%{?rhel} >= 9
    -Dlint=true \
%else
    -Dlint=false \
%endif
```